client/packages/invoices/src/Prescriptions/DetailView/Payments/PaymentsModal.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FC, ReactElement } from 'react';
+import { DialogButton } from '@common/components';
+import { Grid, TextField } from '@openmsupply-client/common';
+import { useDialog } from '@common/hooks';
+import { useTranslation } from '@common/intl';
+import { usePrescription } from '../../api';
+
+interface PaymentsModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+  handleConfirm: () => void;
+}
+
+export const PaymentsModal: FC<PaymentsModalProps> = ({
+  isOpen,
+  onClose,
+  handleConfirm,
+}): ReactElement => {
+  const t = useTranslation();
+  const { Modal } = useDialog({ isOpen, onClose, disableBackdrop: true });
+
+  const {
+    query: { data },
+  } = usePrescription();
+
+  const fields = [
+    {
+      label: t('label.total-to-be-paid'),
+      type: 'number',
+      value: data?.pricing.totalAfterTax,
+    },
```

Comment:
This is just info right, not something the user can edit?  Perhaps it should just be a text label rather than an input to make that clear? At the very least the current input should be `disabled` I'd say.

Reply:
Oh, I should mention, if you do keep it as an input, you probably want to use the "CurrencyInput" component

Reply:
Good point about disabled state. Adding that in! Going to go with `CurrencyInput` passed `InputWithLabelRow` as the rest of the fields are going to be inputs too